sources/packages/api/src/database/entities/assessment-status.type.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Possible status for an Assessment, when the Application_status is in Assessment
+ */
+export enum AssessmentStatus {
+  /**
+   * An Assessment is required
+   */
+  required = "Required",
+  /**
+   * An Assessment is not required, not used in our current workflow but having it as an placeholder
+   */
+  notRequired = "Not Required",

```

Comment:
IMO, if it is something that we would likely have in the future we can leave it here, but I am not sure if it is the case.
@abschwenker, is there any possible scenario where the status "Not Required" would be used?

Reply:
I believe that I misunderstood the conversation that we had in the past. My understanding was that the sub statuses would be the same because they would be following a similar flow. IMHO, they must represent the flow (or what it will probably be) and they can evolve differently. In the future, if we keep states that are not used it could cause confusion, but the comments added will minimize it.
Question, if we found that we need a new status for assessment, for instance, are we going to update all the others ones?

Reply:
Depending what they evolve to they may need to be maintained separately. At this point I see a use case for each of the status's. For this PR and right now we can go under the assumption we will be using these 4 states for each status. However future changes may prove me wrong.